examples/automm/kaggle_feedback_prize/README.md

```diff
@@ -49,10 +49,10 @@ Under the hood, the following process is done when `data.text.normalize_text` is
+    '"Congestion was down\xa0 60\xa0 percent\xa0 in\xa0 the\xa0 capital\xa0 of\xa0 france\xa0 after\xa0 fivedays\xa0 of\xa0 intensifying\xa0 smog."smog\xa0 by\xa0 meaning\xa0 pollution\xa0 went\xa0 down\xa0 just\xa0 60\xa0 percent\xa0 in\xa0 five\xa0 days. Thats\xa0 a\xa0 great\xa0 advantage\xa0 just\xa0 by\xa0 limiting\xa0 car\xa0 usage. In\xa0 source\xa0 number 1\xa0 explains that " Passenger\xa0 cars\xa0 are\xa0 responsible\xa0 for\xa0 12\xa0 percent\xa0 of\xa0 greenhouse\xa0 gas emissions\xa0 in\xa0 Europe.. and up\xa0 to\xa0 50\xa0 perecnt\xa0 in\xa0 some\xa0 car-intensive\xa0 areas\xa0 in\xa0 the\xa0 Untied States." We\xa0 as\xa0 a\xa0 country\xa0 should\xa0 lower\xa0 that\xa0 and\xa0 the\xa0 best\xa0 way\xa0 is\xa0 to\xa0 limiting\xa0 car usage . Limiting car usage is one of te some advantages to lowering pollution ( greenhouse gas, smog). '
 
     # Example-2 post-normalization
-    '"Congestion was down  60  percent  in  the  capital  of  france  after  fivedays  of  intensifying  smog."smog  by  meaning  pollution  went  down  just  60  percent  in  five  days. Thats  a  great  adavantage  just  by  limting  car  usuage. In  source  number 1  explains that " Passenger  cars  are  responsible  for  12  percent  of  greenouse  gas emissions  in  Europe.. and up  to  50  perecnt  in  some  car-intensive  areas  in  the  Untied States." We  as  a  country  shoud  lower  that  and  the  best  way  is  to  limting  car usuage . Limting car usage is one of te some advantges to lowering pollution ( greenhouse gas, smog). '
+    '"Congestion was down  60  percent  in  the  capital  of  france  after  fivedays  of  intensifying  smog."smog  by  meaning  pollution  went  down  just  60  percent  in  five  days. Thats  a  great  advantage  just  by  limiting  car  usage. In  source  number 1  explains that " Passenger  cars  are  responsible  for  12  percent  of  greenhouse  gas emissions  in  Europe.. and up  to  50  percent  in  some  car-intensive  areas  in  the  Untied States." We  as  a  country  should  lower  that  and  the  best  way  is  to  limiting  car usage . Limiting car usage is one of te some advantages to lowering pollution ( greenhouse gas, smog). '
```

Comment:
Please sanity check if it is correct to edit this, I feel that it might not be correct.

Reply:
@sxjscience please advise

Reply:
I have reverted changes to this `README.md` file.